internal/pkg/agent/cmd/container.go

```diff
@@ -653,8 +653,13 @@ func performGET(cfg setupConfig, client *kibana.Client, path string, response in
 	for i := 0; i < cfg.Kibana.RetryMaxCount; i++ {
 		code, result, err := client.Connection.Request("GET", path, nil, nil, nil)
 		if err != nil || code != 200 {
-			err = fmt.Errorf("http GET request to %s%s fails: %w. Response: %s",
-				client.Connection.URL, path, err, truncateString(result))
+			if err != nil {
+				err = fmt.Errorf("http GET request to %s%s fails: %w. Response: %s",
+					client.Connection.URL, path, err, truncateString(result))
+			} else {
+				err = fmt.Errorf("http GET request to %s%s fails. Response: %s",
+					client.Connection.URL, path, truncateString(result))
```

Comment:
[Suggestion]
I'd rather not rely on the server returning the status code on the error response, it's more reliable using the one we received.

Reply:
OK